redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/cache/RedisCache.java

```diff
@@ -269,46 +142,22 @@ public void invalidateAll() {
         }
     }
 
-    /**
-     * @return The default keys pattern.
-     */
-    protected String getKeysPattern() {
-        return getName() + ":*";
-    }
-
     /**
      * Place the value in the cache.
      *
      * @param serializedKey serializedKey
      * @param value         value
      * @param <T>           type of the value
      */
+    @Override
     protected <T> void putValue(byte[] serializedKey, T value) {
         Optional<byte[]> serialized = valueSerializer.serialize(value);
-        if (serialized.isPresent()) {
-            byte[] bytes = serialized.get();
-            if (expireAfterWritePolicy != null) {
-                redisStringCommands.psetex(serializedKey, expireAfterWritePolicy.getExpirationAfterWrite(value), bytes);
-            } else {
-                redisStringCommands.set(serializedKey, bytes);
-            }
-        } else {
-            redisKeyCommands.del(serializedKey);
-        }
-    }
-
-    /**
-     * Serialize the key.
-     *
-     * @param key The key
-     * @return bytes of the object
-     */
-    protected byte[] serializeKey(Object key) {
```

Comment:
deleting this method is a breaking change

Reply:
the japicmp task had it down as a "source compatible" warning (not an error)

I added them back in anyway, and found the Sonar incantation to stop it complaining